utils/client.go

```diff
@@ -27,7 +30,7 @@ import (
 
 // APICommandSetup is a helper for setting up a generic command using Astarte API.
 // individualURLs must contain the service->variable association.
-func APICommandSetup(individualURLVariables map[misc.AstarteService]string, keyVariable, keyFileVariable string) (*client.Client, error) {
+func APICommandSetup(individualURLVariables map[misc.AstarteService]string, keyVariable, keyFileVariable string, ignoreSSLErrors bool) (*client.Client, error) {
```

Comment:
This also has the additional benefit of allowing to set `ignoreSSLErrors` in a config file (e.g. `astartectl` contexts), since usually if you want to ignore SSL errors on a specific instance you always want to do so